pom.xml

```diff
@@ -414,7 +414,7 @@
       <dependency>
         <groupId>org.roaringbitmap</groupId>
         <artifactId>RoaringBitmap</artifactId>
-        <version>0.9.38</version>
+        <version>1.0.2</version>
```

Comment:
Sorry this should actually be 1.0.5, it looks like there's something wrong with the release notes generation in the library, hence the missing version :/